src/test/java/hudson/tasks/AntTest.java

```diff
@@ -165,9 +186,16 @@ public void testParameterExpansion() throws Exception {
         assertTrue("Missing matrix axis $AX: " + log, log.contains("vAX=is"));
     }
 
+    private AntInstallation configureDefaultAnt() throws Exception {
+        TemporaryFolder tmp = new TemporaryFolder();
```

Comment:
:bug: `TemporaryFolder` should be made a field marked with `@Rule` to ensure it is properly cleaned up.